save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestCreationComponent.kt

```diff
@@ -154,7 +154,7 @@ private fun contestCreationComponent() = FC<ContestCreationComponentProps> { pro
     val (testSuites, setTestSuites) = useState(emptyList<TestSuiteDto>())
     useRequest {
         val testSuitesFromBackend: List<TestSuiteDto> = post(
-            url = "$apiUrl/test-suites/get-by-ids",
+            url = "$apiUrl/test-suites/${props.organizationName}/get-by-ids",
```

Comment:
I guess need to take `contestDto.organizationName`

Reply:
Not really sure there is a difference, but will fix it though

Reply:
Done